NEWS.rst

```diff
@@ -2,6 +2,46 @@
 Release Notes
 =============
 
+----------------------------
+pkgcheck 0.10.18 (2022-11-09)
+----------------------------
+
+Special thanks is given to Sam James, for continues support during all
+development, bringing ideas, testing and improving, and especially proofreading
+and improving all docs, texts and help messages. Every release is better thanks
+to him.
+
+- Network checks: fix wrong attributes shown when same URL is checked (#403,
+  Arthur Zamarin)
+
+- BetterCompressionCheck: new check for suggesting better compression URI for
+  distfile fetch (#483, Arthur Zamarin)
+
+- ExcessiveLineLength: report lines longer than 120 characters (with multiple
+  exception rules) (#480, Arthur Zamarin)
+
+- MissingRemoteIdCheck: new check for suggesting missing remote ids, inferred
+  from HOMEPAGE and SRC_URI (#486, Arthur Zamarin)
+
+- DoCompressedFilesCheck: new check for calling ``doman``, ``newman`` and
+  ``doinfo`` with compressed files (#485, Arthur Zamarin)
+
+- AcctCheck: use ``metadata/qa-policy.conf`` for dynamic id range (#489, Arthur
```

Comment:
```suggestion
- AcctCheck: determine dynamic ID range from repository file ``metadata/qa-policy.conf`` rather than static hardcoded values in pkgcheck (#489, Arthur
```